tao_compiler/mlir/disc/transforms/tests/disc-custom-call-rewriter.mlir

```diff
@@ -0,0 +1,57 @@
+// RUN: disc-opt --disc-custom-call-rewriter -split-input-file %s -o - | FileCheck %s
+
+// CHECK-LABEL: @gemm
+// CHECK-SAME: (%[[INPUT:.*]]: tensor<?x?xf32>, %[[WEIGHT:.*]]: tensor<?x?xf32>)
+func.func @gemm(%input: tensor<?x?xf32>, %weight : tensor<?x?xf32>) -> tensor<?x?xf32> {
+  // CHECK: %[[T0:.*]] = "mhlo.transpose"
+  // CHECK-SAME: %[[WEIGHT]]
+  // CHECK-SAME: permutation = dense<[1, 0]>
+  // CHECK: %[[T1:.*]] = "mhlo_disc.custom_call_v2"(%[[INPUT]], %[[T0]])
+  // CHECK: %[[T2:.*]] = "mhlo.transpose"
```

Comment:
need to check index permutation is correct?

Reply:
done, thanks!